spring-data-mongodb/src/main/java/org/springframework/data/mongodb/core/convert/MongoConverters.java

```diff
@@ -75,7 +76,9 @@ static Collection<Object> getConvertersToRegister() {
 		List<Object> converters = new ArrayList<>();
 
 		converters.add(BigDecimalToStringConverter.INSTANCE);
+		converters.add(BigDecimalToDecimal128Converter.INSTANCE);
```

Comment:
This has no effect, as `BigDecimalToStringConverter.INSTANCE` from above always takes precedence. All BigDecimal end up as String in the database.

Why do we still need `BigDecimalToStringConverter`?